docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md

```diff
@@ -12,41 +12,65 @@ isTocNested: false
 showAsideToc: true
 ---
 
-For on-premises deployments of YugabyteDB universes, you need to import nodes that can be managed by the Yugabyte Platform. This page outlines the steps required to prepare these YugabyteDB nodes for on-premises deployments.
+For on-premises deployments of Yugabyte universes, you need to import nodes that can be managed by the Yugabyte Platform. This page outlines the steps required to prepare these YugabyteDB nodes for on-premises deployments.
 
+## Ports
 
+The following ports must be opened for intra-cluster communication (they do not need to be exposed to your application, only to other nodes in the cluster and the platform node):
+
+* 7100 - Master RPC
+* 9100 - TServer RPC
+* 11000 - YEDIS API
+* 12000 - YCQL API
+* 13000 - YSQL API
```

Comment:
My mistake on this PR - these API port information should move under the next section, the "open to administrators and monitoring software"